content/800-data-platform/05-data-browser.mdx

```diff
@@ -0,0 +1,165 @@
+3. To add a new filter, click **Add a new filter** and repeat the steps above.
+4. To remove a filter, click the **x** button on the right.
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657146172/docs/pdp/data-browser-filters-actions_grtsrh.png" width="450px" alt="Data Browser - add and remove apply filters"/>
+
+**Result**
+
+* The data in the model is filtered based on the conditions you added.
+* The default **None** value in the **Filters** menu changes to the number of filters you added.
+
+### Show and hide fields
+
+You can select which fields to view or hide by using the **Fields** menu.
+
+<Admonition>
+
+**What is a field?**<br /><br />
+
+A **field** is a property of a model which you add in the data model definitions in the Prisma schema file. Depending on the database that you use, a field , such as the `title` field in `model User { title String }`, refers to a **column** in a relational database (PostgreSQL, MySQL, SQL Server, SQLite, CockroachDB) or a **document field** in MongoDB.<br /><br />
+For more information, see [Defining fields](/concepts/components/prisma-schema/data-model#defining-fields).
+</Admonition>
+
+**Steps**
+
+1. Click the **Fields** menu.
+2. Select only the fields you want to see and deselect any fields you want to hide.
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657303721/docs/pdp/data-browser-fields-actions-v02_obw71i.png" width="400px" alt="Data Browser - show and hide fields"/>
+
+**Result**
+
+The model is immediately filtered to exclude the data from any fields you have deselected.
+
+Also, the **Fields** menu shows the number of fields that are currently selected.
+
+### Show and hide records
+
+You can also select to show or skip a specific number of records in the model view.
+
+<Admonition>
+
+**What is a record?**<br /><br />
+
+A **record** refers to a row of data in a table in a relational database (PostgreSQL, MySQL, SQL Server, SQLite, CockroachDB) or a **document** in MongoDB.
+</Admonition>
+
+**Steps**
+
+1. Click the **Showing** menu.
+2. In the **Take** box, specify the maximum number of records that you want the model view to show.
+3. In the **Skip** box, specify how many of the first records you want to hide.
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657303813/docs/pdp/data-browser-records-actions_v02_znqx9x.png" width="450px" alt="Data Browser - Show and hide records"/>
+
+**Result**
+
+The model is immediately filtered to show or hide records based on your selection.
+
+The **Showing** menu indicates how many records are shown out of how many available records are in the model.
+
+## Editing data
+
+In the model view, you can edit data directly in the model cells. You can add new records as well as delete existing records.
+
+You must confirm every edit operation (add, edit, delete). You must confirm any new records that you add or edits of existing records that you make with the **Save change** button. 
```

Comment:
This line sounds a bit inconsistent - what about deleting? WHy is that only in the brackets and not explained again later?

Reply:
Good catch.

Reply:
I've added more explanation around that.